docs: update README.md (#159)
SuteraVR/SuteraVR: README.md

<div align="center">
<img src="https://github.com/SuteraVR/SuteraVR/assets/61668366/c73c4885-7589-4edd-864c-156da428312b" width=300>

**the distributed metaverse platform:**<br>**makes your precious relationships built in metaverse independent of any specific service!**

<img src="https://img.shields.io/badge/Rust-v1.74.0 stable-000000.svg?style=for-the-badge&logo=rust">
<img src="https://img.shields.io/badge/Godot Engine-v4.2 .NET-478CBF.svg?style=for-the-badge&logo=godotengine">
<a href="https://github.com/SuteraVR/.github/blob/main/CONTRIBUTING.md"><img src="https://img.shields.io/badge/Become a-Contributor-A371F7?style=for-the-badge&logo=git-merge&labelColor=363B40"></a>
<a href="https://discord.gg/pTjBHkVQbT"><img src="https://img.shields.io/badge/Join-the community-5865F2?style=for-the-badge&logo=discord"></a>

</div>

---

[日本語版はこちら](./READNE_JP.md)

## 🚨 Attention

This software is still in the **development phase** and has not yet been officially released.

> If you interested in our project, come to [the Discord community](https://discord.gg/pTjBHkVQbT)!

Reviewer: ```suggestion
If you interested in our project, come to [our Discord community](https://discord.gg/pTjBHkVQbT)!
```

> We look forward to your contributions! [Open an issue](https://github.com/SuteraVR/SuteraVR/issues/new/choose)/[Open a PR](https://github.com/SuteraVR/SuteraVR/compare)

Reviewer: リンクの前後はスペース入れると窮屈にならなくていいわよ
```suggestion
We look forward to your contributions! [Open an issue](https://github.com/SuteraVR/SuteraVR/issues/new/choose) / [Open a PR](https://github.com/SuteraVR/SuteraVR/compare)
```

## 🧭 Concept

- **Distributed & Decentralized**

>   - To make the metaverse as certain as reality, it must not become fragile things dependent on specific company and platform. Virtual Reality, forever.

Reviewer: ```suggestion
  - To make the metaverse as certain as reality, it must not become fragile things dependent on specific companies or platforms. Virtual Reality, forever.
```
specific companyに依存してても、specific platformに依存してても、つまりどちらか片方でもだめだからorが適切 (そもそも否定文の中だしね)

## ✨ Design

- **Bloom**

>   - You can create a bloom that works as an independent metaverse platform: play with friends, share worlds and items. All blooms communicate with each other, and be able to access contents of other blooms.

Reviewer: ```suggestion
  - You can create a bloom that works as an independent metaverse platform: play with friends, share worlds and items. All blooms communicate with one another, and have access to the contents of other blooms.
```
上は英文法の指摘で、あと不安なのは
これだと特定のBloomがデータを持つように聞こえないかな……一応分散して持ってるわけで……

Author: hmm...どういう言い回しがいいかな
今日の議題に回してもいいかも？

- **Sandbox Enviroment**
  - Complete data processing within the sandbox environment. Prevent viruses and malicious scripts from executing and influencing the outside world.
- **SuteraVR SDK**

>   - You can upload avatars/worlds/items, and attach functions as you like by coding.

Reviewer: ```suggestion
  - You can upload avatars/worlds/items, and attach functions whatever you coded.
```
